### ISSUE TITLE
Template Name and Description Editing Is Not Accessible-Friendly

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1664]
The edit name and description cancel and confirm buttons have no names so their functions are ambiguous to screen reader users. You also can't edit name and description with just keyboard input.
